pytrip/lib/core.c

```diff
@@ -641,12 +644,62 @@ static PyObject * calculate_lvh_slice(PyObject *self, PyObject *args)
     return PyArray_Return(vec_out);
 }
 
+static void append_intersection(PyObject *list_out, double x, double y, double z){
+    PyObject *list_item; // temporary variable to store point in 3D space (represented as list of 3 floats)
+
+    list_item = PyList_New(3);
+
+    PyList_SetItem(list_item, 0, PyFloat_FromDouble(x));
+    PyList_SetItem(list_item, 1, PyFloat_FromDouble(y));
+    PyList_SetItem(list_item, 2, PyFloat_FromDouble(z));
+
+    PyList_Append(list_out, list_item);
+}
+
+static void calc_intersect_sagittal(PyObject *vec_slice, PyObject *list_out, double depth, int i_0, int i_1){
```

Comment:
add a comment explaining which plane is sagittal - XY, YZ or XZ ?

Reply:
also explain what this method is doing, explaining what is the meaning of parameters